### PR TITLE
Android 13, Xperia 10 IV/V built instructions

### PR DESCRIPTION
**Pull request by Thaodan**
Also fix link to Sony AOSP 11 built instructions.

### CODE REVIEW

**Review by jovirkku — COMMENTED** [review]
A few comments on the text clarity from "new to HADK" point of view.

File: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
+---
+title: Sailfish X Xperia Android 13 Build and Flash
+permalink: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/
+parent: HW Adaptation
+layout: default
+nav_order: 600
+---
+
+# Sailfish OS Hardware Adaptation Development Kit for Sony Xperia 10 IV
```
Comment (rainemak) [inline Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md:9]:
```suggestion
# Sailfish OS Hardware Adaptation Development Kit for Sony Xperia 10 IV and Xperia 10 V
```

File: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+## Building
+
+Please download the latest Sailfish OS HADK (Hardware Adaptation Development Kit) from within [this link](/Develop/HADK).
+
+Please check the requirements for your build host: <https://source.android.com/setup/build/requirements>
+
+Minimum Sailfish OS version for this port is 4.6.0. Using Platform SDK version 4.6.0 is required.
+
+If you are new to HADK, please carefully read the disclaimer on page 1, then **chapters 1 and 2**.
```
[the remark this answers is not in the document]
Comment (Thaodan) [reply Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md:22]: The chapters refer to the HADK mentioned in the first sentence of the section.
I think it's clear that we refer to the HADK here as the sentence starts with "If you are new to HADK..".

File: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+srcks=$ANDROID_ROOT/hybris/droid-configs/installroot/usr/share/kickstarts/Jolla-@RELEASE@-$DEVICE-@ARCH@.ks
+sed -i "s @DEVICEMODEL@ $DEVICE " $srcks
+rpm/dhd/helpers/build_packages.sh --mic
+```
+
+The command above will yield a flashable archive, such as `$ANDROID_ROOT/SailfishOS-release-<version>-<device>-my1/SailfishOS--my1-<version>-<device>-<hw-version>.zip` that you'll use to flash your device as explained in the next section.
+
+## Flashing
+
+You will find the instructions for Xperia 10 IV/V on our website that you can easily refer from (yet use your own built .zip bundle!): <https://jolla.com/how-to-install-sailfish-x-on-xperia-10-ii-on-linux/>
```
[the remark this answers is not in the document]
Comment (Thaodan) [reply Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md:224]: Thanks for the spelling fix.

I fixed the link to refer Xperia 10 IV now however it shouldn't be device specific or contain the device file names in a prerequisites section.
Anyway different task.

File: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+If you want to built the image for Xperia 10 V some the variables would be different,
+these would look like below:
+```
+export DEVICE=xqcd54
+export HABUILD_DEVICE=pdx235
+export FAMILY=zambezi
+```
+
+
+Follow through the **chapter 4** until the end, and **ignore chapter 5**, instead perform the following:
```
[the remark this answers is not in the document]
Comment (Thaodan) [reply Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md:48]: The chapters are relevant to other devices like those which start out as a new port.

File: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+title: Sailfish X Xperia Android 13 Build and Flash
+permalink: Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/
+parent: HW Adaptation
+layout: default
+nav_order: 600
+---
+
+# Sailfish OS Hardware Adaptation Development Kit for Sony Xperia 10 IV
+
+Here you will find instructions how to build Sailfish OS image and flash it to Sony Xperia 10 IV or Xperia 10 V device, which can be used as reference to port any other Sony Xperia Open Device that has Android 13 support.
```
[the remark this answers is not in the document]
Comment (Thaodan) [reply Develop/HW_Adaptation/Sailfish_X_Xperia_Android_13_Build_and_Flash/README.md]: I clarified the section a little better, please look again. The background some devices can be built for more than one Android version and it is about which Android version you can built for the device not what is installed.

**Pull request merged**